lintro/ai/review/orchestrator.py

```
@@ -73,6 +91,254 @@ class _ChunkReviewPartial:
+
+        completed = 0
+        for future in as_completed(futures):
+            chunk_index = futures[future]
+            try:
+                partials[chunk_index] = future.result()
+                completed += 1
+            except ReviewExecutionError:
+                for pending in futures:
+                    pending.cancel()
+                raise
+            except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)